configurations/amp-ad/style/_overides.scss

```diff
@@ -12,6 +12,12 @@ footer {
   }
```

Comment:
shouldn't need to override the color below like this but its a quick workaround until I fix the css in SRC